payment-components/vanilla-js/apps/client/scripts/checkout.js

```diff
@@ -51,14 +51,17 @@ async function getData() {
 
         const components = chargebee.components(componentOptions);
 
+        const onSuccess = (payment_intent, extra) => {
```

Comment:
```
const onSuccess = async (payment_intent) => {
    const url = "http://localhost:8082/submit"
```

In this file Line 1, we have onSuccess 
right now its taking only **payment_intent** we can add **payment_intent**, **extra** over there

Reply:
@cb-amalgeorge `onSuccess()` has global scope while the other two callbacks have been implemented locally within `getData()`. Is this intentional?

Reply:
onSuccess, onError and onPaymentMethodChange are implemented in the same level, no?

Reply:
> In this case, onSuccess, onError and onPaymentMethodChange are implemented in the same level, inside getData, yes?

@cb-amalgeorge see line # 1. Is that a duplicate then?

Reply:
Thanks, @cb-amalgeorge. In the `master` branch, `onSuccess()` is global (line #1), while the other two callbacks are local. I'll move all callbacks to global for clarity, modularity, and uniformity.